tests/test_cases/issue_3239/issue_3239.py

```diff
@@ -0,0 +1,6 @@
+import cocotb
+
+
+@cocotb.test()
```

Comment:
You'll have to add conditions for other simulators that will fail this test.
There are many examples in other tests.

Also, could you rename the test to something more descriptive of what you are testing rather than the issue #? Thanks.